main/zoe/guide/contracts/covered-call.md

```diff
@@ -2,160 +2,227 @@
 
-  // Alice makes an offer and gets an option as the outcome
-const { outcome: optionP, payout: alicePayoutP } = await E(zoe).offer(
-  aliceInvite,
+```js
+const aliceSeat = await E(zoe).offer(
+  creatorInvitation,
   aliceProposal,
-  alicePayments,
+  alicePayment,
 );
 
-const option = await optionP;
+const coveredCall = aliceSeat.getOfferResult()
 ```
 
-This option is a full ERTP payment and can be escrowed and used in other
-contracts. For instance, if Alice is the user to makes the first
-proposal and gets the option (a Zoe invite) in return, she can send it to Bob, who can
-either exercise the call option or sell it in another contract, say, an atomic
-swap:
+The offerResult obtained from the seat is a zoe invitation that serves as the
+covered call she wants.  This invitation is a full ERTP payment and can be
+escrowed and used in other contracts. For instance, Alice can send it to Bob,
+who can either exercise the call option or sell it in another contract, say, an
+atomic swap:
 
 ```js
-const inviteIssuer = zoe.getInviteIssuer();
-const bobExclOption = await inviteIssuer.claim(option);
+const invitationIssuer = E(zoe).getInvitationIssuer();
+const bobExclOption = await invitationIssuer.claim(coveredCall);
+```
+
+Let's imagine that Bob wants to sell the invitation.  He can start a swap
+instance to trade this invitation for bucks.
 
-// Let's imagine that Bob wants to sell the invite for the call option.
-// He can create a swap to trade this invite for bucks.
+```js
 const swapIssuerKeywordRecord = harden({
-  Asset: inviteIssuer,
+  Asset: invitationIssuer,
   Price: bucksR.issuer,
 });
-const bobSwapInvite = await zoe.makeInstance(swapInstallationId, swapIssuerKeywordRecord);
+const bobSwapSeat =
+  await E(zoe).startInstance(swapInstallation, swapIssuerKeywordRecord);
+```
+  
+Bob specifies that he wants to swap the invitation for 1 buck, and escrows
+the covered call invitation. In exchange, he gets a swap invitation he can
+share.
 
-// Bob wants to swap the invite with the same units as his
-// current invite from Alice. He wants 1 buck in return.
+```js
 const bobProposalSwap = harden({
-  give: { Asset: inviteIssuer.getAmountOf(bobExclOption) },
+  give: { Asset: invitationIssuer.getAmountOf(bobExclOption) },
   want: { Price: bucks(1) },
 });
 
 const bobPayments = harden({ Asset: bobExclOption });
+const bobSwapSeat = await E(zoe).offer(bobSwapInvitation, bobProposalSwap, bobPayments);
 
-// Bob escrows his option in the swap and gets an invite as the outcome that he can send to Dave
-const { outcome: bobSwapInviteP, payout: bobPayoutP } = await E(zoe).offer(
-  bobSwapInvite,
-  bobProposalSwap,
-  bobPayments,
-);
-
-const daveSwapInvite = await bobSwapInviteP;
+const daveSwapInvitation = bobSwapSeat.getOfferResult();
 ```
 
-Bob now has a call option, in the form of an invite, ready to sell via an atomic swap contract.
+Bob now has a call option that he can sell via an atomic swap contract.
```

Comment:
Hmm, this was in the original but doesn't seem right. Should we move it up before the atomic swap?

Reply:
Once I moved it closer to where it applied, it looked redundant, so I dropped it.